plugins/kubernetes/api-report.md

```diff
@@ -14,18 +14,31 @@ import { KubernetesRequestBody } from '@backstage/plugin-kubernetes-common';
 import { OAuthApi } from '@backstage/core-plugin-api';
 import { ObjectsByEntityResponse } from '@backstage/plugin-kubernetes-common';
 import { RouteRef } from '@backstage/core-plugin-api';
+import { TableColumn } from '@backstage/core-components';
+import { V1Pod } from '@kubernetes/client-node';
 
 // Warning: (ae-forgotten-export) The symbol "ClusterLinksFormatter" needs to be exported by the entry point index.d.ts
 // Warning: (ae-missing-release-tag) "clusterLinksFormatters" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
 export const clusterLinksFormatters: Record<string, ClusterLinksFormatter>;
 
+// Warning: (ae-missing-release-tag) "CustomisationProps" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export interface CustomisationProps {
+  // (undocumented)
+  customPodTableColumns: TableColumn<V1Pod>[];
+}
+
 // Warning: (ae-missing-release-tag) "EntityKubernetesContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityKubernetesContent: (_props: {
+export const EntityKubernetesContent: ({
+  customisationProps,
+}: {
```

Comment:
Hm, not sure about the verbosity here - I would have gone with just a simple prop named `podTableColumns` directly on the component

```ts
export interface EntityKubernetesContentProps {
  entity?: ...
  podTableColumns?: ...
};

// note that the props aren't unpacked here, but instead inside the body
export const EntityKubernetesContent = (props: EntityKubernetesContentProps) => {
  const { podTableColumns } = props;
```

and then repeat the exact same prop on Router below too (i prefer the duplication, rather than having to invent a type). Remember to unpack the props inside the function there too